data_reader/bandit_reader.py

```diff
@@ -36,25 +36,41 @@ def __init__(
         self.training_query = self._build_query()
 
     def _build_query(self):
+
         return f"""
-        with r as (
+        with immediate_r as (
             select
                 decision_id,
+                decision,
+                mdp_id,
                 metrics,
                 experiment_id
-            from `gradient-decision.gradient_app_staging.rewards`
+            from `{self.rewards_table_name}`
             where DATE(_PARTITIONTIME)
                 between "{self.decisions_ds_start}" and "{self.rewards_ds_end}"
```

Comment:
so the main diff between immediate and delayed is the date range, yeah?

Reply:
the delayed rewards have decision_id and decision as null, immediate rewards have that filled in. that should be how we distinguish them. added an example to the readme.